masterlist.yaml

```diff
@@ -22647,6 +22647,15 @@ plugins:
       - crc: 0x64014310
         util: 'SSEEdit v4.0.4'
 
+  - name: 'MarkekrausChairhammers.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/61304/' ]
+    req:
+      - <<: *SKSE
+        condition: 'version("MarkekrausChairhammers.esp", "1.0.0", >=)'
```

Comment:
@markekraus Is v1.0.0 the initial version of the mod, or are there alphas/betas publicly available that don't require SKSE? If so, we shouldn't need to check for the plugin's version number.

Reply:
I had some alphas distributed to some people who were till to do some testing for me. But I guess we can just require it for all versions and I'll just let the beta testers know.